docs/docs/configuration/providers/gitlab.md

```diff
@@ -26,11 +26,15 @@ The following config should be set to ensure that the oauth will work properly.
 
 Restricting by group membership is possible with the following option:
 
+```
     --gitlab-group="mygroup,myothergroup": restrict logins to members of any of these groups (slug), separated by a comma
```

Comment:
```suggestion
```shell
    --gitlab-group="mygroup,myothergroup"  # restrict logins to members of any of these groups (slug), separated by a comma